lib/geordi/commands/checkout.rb

```diff
@@ -0,0 +1,16 @@
+desc 'checkout', 'Checkout a feature branch using a story title from Pivotal Tracker'
```

Comment:
Please name the command `branch` or `feature-branch` as "checkout" is a rather abstract term with multiple meanings.

```suggestion
desc 'branch', 'Check out a feature branch based on a story from Pivotal Tracker'
```